examples/platform/silabs/MatterConfig.cpp

```diff
@@ -129,6 +155,44 @@ CHIP_ERROR SilabsMatterConfig::InitOpenThread(void)
 }
 #endif // CHIP_ENABLE_OPENTHREAD
 
+namespace {
+    void application_start(void * unused)
```

Comment:
a namespace doesn't require ident for its content. Restyler is probably complaining about this

Reply:
Yes. Updated.